staff_features/resources/step_definitions/resource_merge.rb

```diff
@@ -0,0 +1,179 @@
+# frozen_string_literal: true
+
+Given 'two Resources A & B have been created' do
```

Comment:
I wonder if it would be better to have a helper function for creating resources. this code appears in number of places, and if anything about this process changes a lot of tests will need to be updated.

Reply:
I have thought of this since the beginning.
A step like `a Resource has been created`, could be considered a function and it's intended to be re-usable anywhere a resource is required.
This specific step (`two Resources A & B have been created`), is an edge case, in which we require two resources and it is currently intended for use only in this feature.
In the future, if we need multiple resources with differing underlying values, maybe we can create an abstraction for creating records.
However, the concern of future changes applies everywhere on these tests (and any tests in general). For example, here we are using Capybara, and we check for HTML elements in all step definitions.